bats/tests/containers/switch-engines.bats

```diff
@@ -0,0 +1,56 @@
+# Test case 20
+
+setup() {
+    load '../helpers/load'
+    RD_CONTAINER_ENGINE=moby
+}
+
+teardown_file() {
+    load '../helpers/load'
+    run rdctl shutdown
+    assert_nothing
+}
+
+switch_container_engine() {
+    local name=$1
+    RD_CONTAINER_ENGINE="${name}"
+    run rdctl set --container-engine.name="${name}"
+    assert_success
+    wait_for_container_engine
+}
+
+pull_nginx() {
+    ctrctl run -d -p 8085:80 --restart=no nginx
```

Comment:
I think we should start 2 containers, one with `--restart=no` and one with `--restart=always`, and when you switch away, and then back to it, verify not only that the `--restart=no` one is gone, but also that the `--restart=always` has been started again.

Reply:
done